### ISSUE TITLE
[a11y-defect-1]:Alert and status messages aren't announced without receiving focus.

### ISSUE THREAD

**Issue by SarahKay8** [department-of-veterans-affairs/va.gov-team#65505]
### Point of contact

SK (Sarah Koomson) 

### Severity level

1, Launchblocker. Critical. Must be fixed before launch.

### Details

...
After selecting yes or no on “are you interested in using tele-counseling?” Paragraph “ Please tell us the time you prefer to meet with your counselor. We’ll make every effort to assign a counselor who is available to meet during your preferred hours.” Is being skipped by screenreaders.
Screen reader users will not know this paragraph exists as it is sandwiched between form controls. Using the tab key will only surface the link. Can put this information into an additional info component beneath “When are the  best times to meet with your counselor?” So information is not skipped. 

### Reproduction steps

1. [Link to page](https://staging.va.gov/careers-employment/vocational-rehabilitation/apply-vre-form-28-1900/communication-preferences) 
2. Select "yes" or "no" on Are you interested in using Tele-counseling?
3.  Confirm that AT announces fieldset underneath paragraph and skips the paragraph 

### Proposed solution or next steps

...
Can put this information into an `additional info` component beneath “When are the  best times to meet with your counselor?” So information is not skipped. 

### References, articles, or WCAG support

...

https://github.com/department-of-veterans-affairs/va.gov-team/assets/102324990/0ccdb340-0ff7-41d9-9485-53ad8056d7d7

### Type of issue

- [ ] Axe-core
- [X] Screenreader | assistive tech & device support
- [ ] Keyboard
- [ ] Zoom
- [ ] Color, typography, & visual elements
- [ ] Components and pattern standards
- [ ] Content organization
- [ ] Focus
- [ ] Headings
- [ ] Content style guide
- [ ] Link and button labels
- [ ] Semantic HTML
- [ ] Markup and metadata
- [ ] Non-text content and media
- [ ] Overall user experience
- [ ] Cognitive | user flows & navigating
- [ ] URL standards
- [ ] Something else

**Comment by zack** [comment]
This is low lift and I have an MVP done. I do need to know what text we want for the 'more information' trigger before I can open a PR.

**Comment by sortizsh** [comment]
we will take this to CAIA office hrs

**Comment by SarahKay8** [comment]
@sortizsh! Apologies on delay was taking PTO. Updated ticket with new preferred solution.

**Comment by sortizsh** [comment]
CAIA convo here https://dsva.slack.com/archives/C01K37HRUAH/p1694805117702029

**Comment by zack** [comment]
Hi team,

After chatting with some members of the Platform team, it appears that none of the suggested solutions are possible within the forms system that we have. We can move the description, but it's not possible to have it read out before/when the user navigates to the checkbox group. The best we can do with the existing system is to have it read off when the first (or we could add it to all) checkbox is selected, which is the current implementation.

The only other solution I can think of would be to add `tabIndex="0"` to the description text (either in its current or newly proposed location), but I understand that's generally considered a bad practice for accessibility, and I'm not sure how the benefits balance out here.

cc: @SarahKay8

**Comment by SarahKay8** [comment]
Thank you @zack! Glad we were able to chat about this today [Link to slack thread](https://dsva.slack.com/archives/CBU0KDSB1/p1695672339863949)
